geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZUnionStoreExecutor.java

```diff
@@ -52,6 +57,11 @@ public RedisResponse executeCommand(Command command, ExecutionHandlerContext con
       return RedisResponse.error(ERROR_SYNTAX);
     }
 
+    // Rough validation so that we can use numKeys to initialize the array sizes below.
```

Comment:
`numKeys` is not used to initialize the arrays, despite this comment implying it will be.

Reply:
Also too much ^Z